mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = (req, res) => {
+  Order.remove({ _id: req.params.orderId }) 
+    .exec((error, result) => {
```

Comment:
Inconsistent error handling. In other parts of the code, error handling is done using `errorHandler` function, but here it's not used. It should be consistent throughout the code.